modules/userId/eids.js

```diff
@@ -81,6 +81,11 @@ const USER_IDS_CONFIG = {
   'netId': {
     source: 'netid.de',
     atype: 1
+  },
+  // sharedId
+  'sharedId': {
+    source: 'rubiconproject.com',
```

Comment:
```suggestion
    source: 'sharedid.org',
```
Need to confirm with Garrett, but I believe we don't want rubicon as the source.